### ISSUE TITLE
Problem with creating Invoice ({"error":"No derivation strategy are available now for this store"})

### ISSUE THREAD

**Issue by Uter1007** [btcpayserver/btcpayserver#64]
I've used the One-Click Deployment of https://github.com/btcpayserver/btcpayserver-azure. When I am configuring my store (& Derivation Scheme) I am using my xpub of blockchain.info account. Everything looks good so far, but as soon I am creating an invoice I will get the Error:

`{"error":"No derivation strategy are available now for this store"}`

Cheers,
Christoph

**Comment by NicolasDorier** [comment]
Yes, you need to go to your store settings and configure your destination wallet.

Take a look at https://www.youtube.com/watch?v=xh3Eac66qc4

If you have a ledger wallet, you can easily configure BTCPay to use it.

**Comment by Uter1007** [comment]
Don't get it, sorry.  I would like to use my blockchain.info account (so I am used the xpub of it and I can verify that the derivation scheme looks correct). The store is configured. 

Wouldn't find any setting that I am missing or is only offline wallets (ledger / trezor) supported ?

I am used the youtube video already as my guide, but the only difference is the mycelium \ blockchain difference. At the invoice step the error pop ups. 

Cheers,
Christoph

**Comment by NicolasDorier** [comment]
If you don't know what an xpubkey is, the best is that you use a Ledger so you don't have to know.

**Comment by Uter1007** [comment]
Afaik I am using the xpubkey correctly - I am using mycelium gear (https://admin.gear.mycelium.com/) in the same way that I would like to use BTCPay.  I am pretty sure I am using the correct key, otherwise the 9 derived keys wouldn't match with my addresses in blockchian.info or do I miss something here ? 

Cheers,
Christoph

**Comment by NicolasDorier** [comment]
ah so you did set up the xpubkey ?

**Comment by NicolasDorier** [comment]
If those match you verified and hit save, then you should be able to generate invoice

**Comment by Uter1007** [comment]
I did :( Still think that something went wrong on my machine - I did the one-click deployment (maybe I fucked something up there). Can I provide you some screenshots for further analysis ? 

Thx for your support so far.

Cheers,
Christoph

**Comment by NicolasDorier** [comment]
Can you come on http://slack.forkbitpay.ninja/ ? You can give me access to your BTCPay instance and account on the server so I can try myself.

**Comment by NicolasDorier** [comment]
My guess is that you created two stores and when creating invoice you are choosing the wrong store

**Comment by NicolasDorier** [comment]
you found out your issue?

**Comment by Uter1007** [comment]
sorry haven't got the time the last couple of days. I will try again on weekend. thx for your patients.

**Comment by Uter1007** [comment]
since newest version, everything works as expected ! thx again
